AutoFinance.Broker/InteractiveBrokers/ITwsObjectFactory.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed under the Apache License, Version 2.0.
+
+namespace AutoFinance.Broker.InteractiveBrokers.Controllers
+{
+    using AutoFinance.Broker.InteractiveBrokers.Wrappers;
+    using IBApi;
+
+    /// <summary>
+    /// Interface for TwsObjectFactory
+    /// </summary>
+    public interface ITwsObjectFactory
```

Comment:
do you really want to mock the factory?

Reply:
In my use case, I pass the factory object through all my services since they contain all the objects related to the TWS connection.

I could create another class that wraps that information but I guess someone else might have the same need.